functions/Get-SqlMaxMemory.ps1

```diff
@@ -61,20 +60,7 @@ Find all servers in CMS that have Max SQL memory set to higher than the total me
 	{
 		$collection = @()
```

Comment:
I was mistaken in keeping everything in a $collection then releasing it at the end. We are now moving all commands to streaming the output. Please remove this variable and I'll explain more in detail below.